mailpoet/lib/Automation/Engine/Storage/WorkflowStorage.php

```diff
@@ -73,9 +121,24 @@ public function getActiveTriggerKeys(): array {
 
   /** @return Workflow[] */
   public function getActiveWorkflowsByTrigger(Trigger $trigger): array {
-    $table = esc_sql($this->table);
+    $workflowTable = esc_sql($this->workflowTable);
+    $versionTable = esc_sql($this->versionsTable);
     $query = (string)$this->wpdb->prepare(
-        "SELECT * FROM $table WHERE status = %s AND trigger_keys LIKE %s",
+        "
+SELECT
+       workflow.*,
+      version.id AS version_id,
+      version.steps,
+      version.trigger_keys
+FROM
+     $workflowTable AS workflow
+        LEFT JOIN
```

Comment:
This is a small detail, but we may use just `JOIN` (inner join, I mean) as workflow without a record in `workflow_versions` would be invalid.

Reply:
Addressed in [2ebd723](https://github.com/mailpoet/mailpoet/pull/4289/commits/2ebd7236d596787009393ed244de12682ab52e40)